HandmadeMath.h

```diff
@@ -2644,3 +2645,7 @@ operator*=(hmm_quaternion &Left, float Right)
 #endif /* HANDMADE_MATH_CPP_MODE */
 
 #endif /* HANDMADE_MATH_IMPLEMENTATION */
+
+#ifdef __clang__
+#pragma GCC diagnostic pop
```

Comment:
not sure this is the right place for the pop - the push above happens in the `#ifndef HANDMADE_MATH_H` section, so the pop probably should be at the end of that section (otherwise I think it could happen that one only gets the pop, if the header for some reason is included twice).